### PR TITLE
Fix #5, add links and basic descriptions to README

### LINKED ISSUE #5: Link to other documents
The README currently contains minimal information.  I propose we create links to the other documents and provide a brief outline of what they each cover.

### PR DESCRIPTION
**Pull request by aaaaalbert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
-# lab-guidelines
-How-to guides for SSL projects and documents
+# Secure Systems Lab Guidelines
+
+Welcome to the Secure Systems Lab (SSL) Guidelines repo! Below you find
```
Comment (vladimir-v-diaz) [inline README.md:3]: What about `Below you can find ...` ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+relevant guides and how-tos for working and collaborating in the lab.
+
+* On-boarding -- You are new to SSL and its operations?
+  The [on-boarding guide](on-boarding.md) helps you get up to speed.
+* Starting new projects -- Follow our [new projects guide](newproject.md)
+  to start your project on the right foot.
+* Reporting progress -- Document your achievings and share your insights
```
Comment (vladimir-v-diaz) [inline README.md:10]: achievements ?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+  by following the [progress report guide](progressreport.md).
+* Writing LaTeX documents -- Clear communication is key at SSL, and
+  our [LaTeX guide](latexdocuments.md) helps you master science's Number 1
```
Comment (vladimir-v-diaz) [inline README.md:13]: Why is `Number 1` capitalized?

Comment (vladimir-v-diaz) [inline README.md:13]: Consider instead "an academic researcher's number 1  document preparation tool."

**Review by vladimir-v-diaz — APPROVED** [review]
LGTM!  I just have some minor nitpicks.

File: README.md (hunk 1 above)
Comment (aaaaalbert) [reply README.md:3]: Sure, fine by me.

File: README.md (hunk 2 above)
Comment (aaaaalbert) [reply README.md:10]: Whew, how did I miss that!

File: README.md (hunk 3 above)
Comment (aaaaalbert) [reply README.md:13]: I was thinking of something among the lines of "The Number 1 Used Cars Dealer" which looked typographically idiomatic to me. I'm fine either way.

Comment (aaaaalbert) [reply README.md:13]: No problem. My intention was to keep the overview *casual*.

**Comment by aaaaalbert** [comment]
Thanks for your comments! I'll close this PR, as it has conflicts with `master`. See #8 instead.

File: README.md (hunk 3 above)
Comment (vladimir-v-diaz) [reply README.md:13]: Ahh.  I'll try to loosen my stiff collar for the next round of reviews!

**Pull request closed without merging**